docs/source/conf.py

```diff
@@ -18,7 +18,7 @@
 #
 import sphinx_rtd_theme
 
-sys.path.insert(0, os.path.abspath(".."))
+sys.path.insert(0, os.path.abspath("../../src"))
```

Comment:
:raised_hands: Nice!!

Quick question - trying to catch up on our progress with the docs.. just wondering about the `extract_transform` and `load` stuff. Do those get addressed with this change as well or is the plan for us to tackle them later?

Reply:
Can apply the same for `extract_transform` also, `extract_wiki.py` and `process_wiki.py` have some helper functions too.
not sure about `load`.